crates/types/Cargo.toml

```
@@ -28,7 +28,7 @@ ethereum-types = { workspace = true }
 futures = { workspace = true }
 cdn-proto = { workspace = true }
 reqwest = { workspace = true }
-
+serde-inline-default = "0.1"
```

can we move this to the workspace (maybe with the latest version, `= "0.2"` unless that breaks something) and do

```suggestion
serde-inline-default = { workspace = true }
```

---

Oh, that's embarassing, didn't realize I duplicated the import in two spots!